_docs/_latest/setup/manual-gcp-deployment.md

```diff
@@ -0,0 +1,141 @@
+        
+1. Run the steps in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
+1. Create [firewall rules](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/install/gcp/installer/forseti_server_installer.py)
+
+### Configuration
+
+Forseti server configuration and rule files are configured and stored in
+Cloud Storage. For more information, see
+[Configuring Forseti]({% link _docs/latest/howto/configure/configuring-forseti.md %}).
+
+### G Suite Integration
+
+To integrate G Suite, you'll need to enable domain-wide delegation in G Suite.
+For more information, see [Enabling G Suite Google Groups Collection](https://forsetisecurity.org/docs/howto/configure/gsuite-group-collection.html)
+
+## Create a Database
+
+### Create a Cloud SQL instance.
+
+To create a For information about the specifications needed for a Cloud
+SQL instance, see the [Cloud SQL template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/cloudsql/cloudsql-instance.py).
+
+## Deploy a client VM
+
+### Create a Forseti client service account
+
+Create a client service account by running the following command:
+
+```
+forseti-client-gcp-#######@fooproject.iam.gserviceaccount.com
+```
+
+Where `#######` is a random alphanumeric unique identifier that must be
+accepted when used for the Cloud Storage bucket name.
+
+### Assign roles
+
+For information about the roles you need to assign to the Forseti server
+service account, see
+[the client service account]({% link _docs/latest/concepts/service-accounts.html#the-client-service-account %}).
+
+### Create a Forseti client VM Instance
+
+* For information about the specifications needed for a Forseti server
+  VM instance, see the [Compute Engine deployment template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
+* When you create the instance, bind the server service account to the VM instance
+* [Enable computeOS login](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/install/gcp/installer/util/gcloud.py)
+
+### Install the Forseti client
+
+To install the Forseti client, follow the steps below:
+
+1. SSH into the client VM
+1. Run the following command to become an Ubuntu user:
+
+        sudo su ubuntu
+        
+1. `git clone` from the [forseti repo](https://github.com/GoogleCloudPlatform/forseti-security), and check out the [latest release](https://github.com/GoogleCloudPlatform/forseti-security/releases) by their tags.
+
+        git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+        git checkout tags/<tag_number>
+        
+1. Run the steps in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
```

Comment:
This needs explanation and or specific descriptions.

Reply:
Updated